Xamarin.Forms.Maps/Pin.cs

```diff
@@ -65,8 +65,13 @@ public object MarkerId
 			}
 		}
 
+		[Obsolete("This event is obsolete. Please use MarkerClicked and/or InfoWindowClicked instead.")]
```

Comment:
The obsolete message should include the version, e.g. "This event is obsolete as of version 4.1.0".

Reply:
Just didn't know when I made the PR what version this would be making it into.